Pipfile

```diff
@@ -9,6 +9,7 @@ requests = ">=2.20.0"
 commonmark = "*"
 num2words = "*"
 pluginbase = "*"
+black = "*"
```

Comment:
This should likely be a dev-package, and may have been installed by mistake.

Reply:
Make sure that you run `pipenv install --dev` to get the development packages as well as the rest, so that during development you can use what you need.